jormungandr-integration-tests/src/common/configuration/node_config_builder.rs

```diff
@@ -27,7 +27,7 @@ impl NodeConfigBuilder {
     pub fn new() -> NodeConfigBuilder {
         let rest_port = super::get_available_port();
         let public_address_port = super::get_available_port();
-        let listen_address_port = super::get_available_port();
+        let _listen_address_port = super::get_available_port();
```

Comment:
Can we just remove that? Seems like this unused statement doesn't have any side-effects.

Reply:
Yeah, let's do that, I chose the `_` because it was a less aggressive  change